conference.js

```diff
@@ -1202,11 +1198,14 @@ export default {
             items[key] = param[1];
         }
 
-        this.e2eeKey = items.e2eekey;
+        if (typeof items.e2eekey !== undefined) {
```

Comment:
A note not related to this exact PR but for the whole onHashChange thing. I don't see why this need to be in conference.js. Why not put it where we process all parameters coming from the URL? Also why you are not reusing the URL functions that we already have implemented to parse the URL?

Reply:
Beccause this is not the final solution but a stopgap way to test it out.

Reply:
So?